src/server/main_service.cc

```diff
@@ -1123,6 +1124,11 @@ void Service::Shutdown() {
 
   shard_set->PreShutdown();
   shard_set->Shutdown();
+
```

Comment:
Lets introduce `SearchFamily::Shutdown()` and move it there -no need to leak the implementation into main_service.cc